WooCommerce/Resources/Images.xcassets/woo-store-switcher.imageset/Contents.json

```diff
@@ -0,0 +1,12 @@
+{
+  "images" : [
+    {
+      "idiom" : "universal",
+      "filename" : "iOS-store-switcher.pdf"
+    }
+  ],
+  "info" : {
+    "version" : 1,
+    "author" : "xcode"
+  }
+}
```

Comment:
I think we need to check "preserve vector data" here:

<img width="255" alt="screen shot 2019-02-18 at 9 31 25 am" src="https://user-images.githubusercontent.com/154014/52961653-0f1e5080-3361-11e9-81cb-2a83903b6ffd.png">

Reply:
Nice catch